scipy/stats/_continuous_distns.py

```diff
@@ -419,8 +419,10 @@ def fit(self, data, *args, **kwds):
         # Override rv_continuous.fit, so we can more efficiently handle the
         # case where floc and fscale are given.
 
-        f0 = kwds.get('f0', None)
-        f1 = kwds.get('f1', None)
+        f0 = (kwds.get('f0', None) or kwds.get('fa', None) or
+              kwds.get('fix_a', None))
```

Comment:
Ah, missed that. Then this is fine. ``rv_continuous.fit`` docstring still needs a tweak.